fibermiddleware.go

```diff
@@ -0,0 +1,84 @@
+package glogger
```

Comment:
We could create a `support` folder, and add the supported framework under this folder. Both Gorilla and Fiber. What do you think?

Reply:
Maybe `middleware` is more explicative?

Reply:
I agree with this suggestion

Reply:
Ok with middleware

Reply:
Moved middleware related files into a new package called `middleware`